util/chplenv/defaults.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+"""
+Check .chplconfig file and environment variables for default value overrides
+"""
+from __future__ import print_function
+import os
+import sys
+from sys import stderr
+
+chplenv_dir = os.path.dirname(__file__)
+sys.path.insert(0, os.path.abspath(chplenv_dir))
+
+# Global dictionary that will be populated w/ .chplconfig file contents
+chplconfig = {}
+
+# Globally track if we've parsed already
+parsed = False
+
+# Valid options for override by .chplconfig file
+CHPLS = [
```

Comment:
Why do you need this array? Can't you just stuff whatever variables are in the .chpconfig file into the dictionary? My concern is that it'll be easy to miss updating this list if a new var is added.